### PR TITLE
Chore: Ignore `chrome` on CI

### LINKED ISSUE #2031: [Bug] Compat-API tests fail fairly often on Windows and Azure Pipelines [0.2]
I believe it's because there are so many tests with different configurations. This causes:

- Fully new instances of Chrome to be created
- Lots of new tests servers

From what @sarvaje told me his machine runs like a 🐌 and is on 🔥 when running these tests (and he has a good one...)

The solutions I can think of are:

- Split the tests in multiple files so ava doesn't try to execute all simultaneously
- Add the option to have a single server when testing on CI (I though I opened an issue for this but can't find it right now)

@webhintio/core thoughts?

### PR DESCRIPTION
**Pull request by molant**
## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/webhintio/hint)
- [x] Followed the [commit message guidelines](https://webhint.io/docs/contributor-guide/getting-started/pull-requests/#commit-messages)

For non-trivial changes, please make sure you also:

- ~~Added/Updated related documentation.~~
- [x] Added/Updated related tests.

## Short description of the change(s)

Fix #2031

### CODE REVIEW

File: packages/hint-compat-api/tests/css-next.ts [hunk 1]
```diff
@@ -4,6 +4,8 @@ import readFile from 'hint/dist/src/lib/utils/fs/read-file';
 import { HintTest } from '@hint/utils-tests-helpers/dist/src/hint-test-type';
 import * as hintRunner from '@hint/utils-tests-helpers/dist/src/hint-runner';
 
+import { ignoredConnectors } from './fixtures/ignored-connectors';
```
Comment (sarvaje) [inline packages/hint-compat-api/tests/css-next.ts]: Is `fixtures` the best place to have this script?

Comment (molant) [reply packages/hint-compat-api/tests/css-next.ts]: I tried with utils and a a complained about not having tests in that folder. I'll take a look see if there's another convention to use.

Comment (antross) [reply packages/hint-compat-api/tests/css-next.ts]: From https://github.com/avajs/ava/blob/f4b2d196e813cf3549e19703e5a79ebc988fb8ca/docs/05-command-line.md

> Directories are recursed, with all *.js files being treated as test files. Directories named `fixtures`, `helpers` and `node_modules` are always ignored. So are files starting with `_` which allows you to place helpers in the same directory as your test files.

**Comment by molant** [comment]
> I think your approach is the most practical solution to our immediate problem of flaky tests.

I actually started separating tests by files but it wasn't really practical 😞

File: packages/hint-compat-api/tests/css-next.ts (hunk 1 above)
Comment (molant) [reply packages/hint-compat-api/tests/css-next.ts]: I've renamed the file to `_ignored-connectors.ts` and moved it with the tests.

**Pull request merged**